scripts/intervals_parser.py

```diff
@@ -0,0 +1,244 @@
+from datetime import date
+import gcsfs
+import pandas as pd
+
+import hydra
+
+from pyliftover import LiftOver
+
+from pyspark.sql import dataframe
+import pyspark.sql
+import pyspark.sql.types as t
+import pyspark.sql.functions as f
+from pyspark.sql import SparkSession
+
+
+class LiftOverSpark:
```

Comment:
I would move this class to an utils module.

Reply:
Yes. Most likely I'm going to split the parsers out as well into separate modules. It' getting way too long.